src/main/resources/view/MainWindow.fxml

```diff
@@ -40,6 +40,8 @@
           </padding>
           <!-- Contact list -->
           <VBox fx:id="personList" styleClass="pane-with-border" minWidth="400" prefWidth="400">
+            <StackPane VBox.vgrow="NEVER" fx:id="searchBoxPlaceholder" styleClass="pane-with-border">
```

Comment:
This can be a self-closing tag.

Also, prefer to keep the `fx:id` to be the first attribute